### ISSUE TITLE
polybar-6 font and hiDpi issues on Arch

### ISSUE THREAD

**Issue by psiho** [adi1090x/polybar-themes#67]
The bar looks amazing but I'm having issues configuring it.

1) Some of my icons were not showing up. I discovered font-1 was named "Siji" and my font-list showed "Wuncon Siji". The problem went away after changing config.ini accordingly.

2) All looked ok but tiny on my hiDPI screen. Setting `dpi = ${xrdb:Xft.dpi:-1}` in [bar/main] increased only main font. Icons stayed the same (tiny). Also, the bar was clipped at the top side. I can't seem to solve either. Playing with height, font sizes... nothing changes.

**Comment by psiho** [comment]
Sorted this out, but I would like to have known this theme is not compatible with hiDPI screens from the start.

The first issue stays, font names are completely wrong for my system (installed supplied fonts). Siji name is different, Terminus I can't even find in the fonts folder. I changed namings to fit my system/fonts.

The second issue is with Siji font icons which are bitmaps and cannot be sized. I failed to find a proper TTF version of it which means all icons in config files need to be replaced (after changing icon font to TTF Material design icons).

Lastly, I had to carefully choose Iosevka Nerd Font sizing and vertical spacing so my bar looks good even after changing height of the bar.

So basically (other than icons changed), important parts of the config to change were:

```
font-0 = "DejaVuSans:size=9;5"
font-1 = "MaterialDesignIconsDesktop:style=regular:size=10;6"
font-2 = "Iosevka Nerd Font:style=Medium:size=10;4"

height = 32
```
... then disable right-click type changer or update all configs.
